### PR TITLE
giving a new pa1k4-out position and add st1k4 veto

### PR DESCRIPTION
**Pull request by tongju12**
## Mingfu gave new OUT positions and add two more targets, but all other targets positions are not ready yet
Adding St1K4 veto here (should have a different PR but since it was urgent, so I add this into same PR with PA1K4
Edit one PV in Sp1k4-flow sensor to make up previous Nick W PR

Adding St1K4 veto here
Add two more targets in Enum of pa1k4, update PA1K4 OUT position
## Motivation and Context

St1k4 is request from James, to protect down stream devices while TMO stopper out
Mingfu asked to add two more targets

## How Has This Been Tested?

https://jira.slac.stanford.edu/browse/ECS-5586

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: plc-tmo-motion/tmo_motion/POUs/PRG_3_PMPS_POST.TcPOU [hunk 1]
```diff
@@ -23,17 +23,17 @@ bM1K3Veto := NOT PMPS_GVL.stCurrentBeamParameters.aVetoDevices[PMPS.K_Stopper.MR
 
 // Adding TXI MR1K3 veto in
```
Comment (nrwslac) [inline plc-tmo-motion/tmo_motion/POUs/PRG_3_PMPS_POST.TcPOU:24]: What does this comment mean? I see MR1K1 being added for veto not MR1K3. @tongju12

Comment (nrwslac) [reply plc-tmo-motion/tmo_motion/POUs/PRG_3_PMPS_POST.TcPOU:24]: Should MR1K3 be added as well?

File: plc-tmo-motion/tmo_motion/DUTs/ENUM_Sample_Calibration_States.TcDUT [hunk 2]
```diff
@@ -11,7 +11,9 @@ TYPE ENUM_Sample_Calibration_States :
     Target2 := 3,
     Target3 := 4,
     Target4 := 5,
-    Target5 := 6
+    Target5 := 6,
```
Comment (nrwslac) [inline plc-tmo-motion/tmo_motion/DUTs/ENUM_Sample_Calibration_States.TcDUT:14]: This one off naming stresses me out.

File: plc-tmo-motion/tmo_motion/POUs/PRG_3_PMPS_POST.TcPOU (hunk 1 above)
Comment (tongju12) [reply plc-tmo-motion/tmo_motion/POUs/PRG_3_PMPS_POST.TcPOU]: I remove the comment as we discussed. After MR1K3 veto really activate later, I will add it then.

**Review by nrwslac — APPROVED** [review]
lgtm

**Pull request merged**